jaxrs-spec/src/main/asciidoc/chapters/appendix/_bibliography.adoc

```diff
@@ -21,11 +21,11 @@
 - [[[bib7,7]]]  J.C. Gregorio and B. de hOra. The Atom Publishing Protocol. Internet Draft, IETF, March 2007. See
                https://bitworking.org/projects/atom/draft-ietf-atompub-protocol-14.html.
 
-- [[[bib8,8]]]  G. Murray. Java Servlet Specification Version 2.5. JSR, JCP, October 2006. See
-               https://java.sun.com/products/servlet.
+- [[[bib8,8]]]  Jakarta™ Servlet Specification, Version 4.0. Available at:
+               https://jakarta.ee/specifications/servlet/4.0
 
-- [[[bib9,9]]]  R. Chinnici, M. Hadley, and R. Mordani. Java API for XML Web Services. JSR, JCP, August 2005.
-               See https://jcp.org/en/jsr/detail?id=224.
+- [[[bib9,9]]]  Jakarta™ XML Web Services Specification, Version 2.3. Available at:
+               https://jakarta.ee/specifications/xml-web-services/2.3
```

Comment:
This link results in a 404. I would be fine with simply removing the URL if there isn't an official site yet.

Reply:
The migration of JAX-WS to Jakarta is decided, but has not started. Hence there yet is not such URL and I doubt it won't change in near future. So it would be best to keep to old one (as it is still correct) until the new one is finally defined.